### DISCUSSION TITLE
Clarifications on pkgdown workflows

### DISCUSSION THREAD

**Discussion by maelle** [cynkra/dm#1948] [General]
@krlmlr reg #1064 to follow https://pkgdown.r-lib.org/dev/articles/how-to-update-released-site.html I need some clarifications.

- we need a workflow dispatch: do I add it to https://github.com/cynkra/dm/blob/main/.github/workflows/pkgdown.yaml

**Comment by maelle** [comment]
To make the workflow dispatch able to replace the released website do you agree we need to replace the lines 

```
if: github.event_name != 'push'
```

with 

``` 
if: github.event_name != 'pull_request'
```

**Comment by krlmlr** [comment]
The pkgdown deployment workflow should be triggered for all branches named `docs*` ?
